nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -6,17 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can use the Node Maintenance Operator to place nodes in maintenance mode. This is a standalone version of the Node Maintenance Operator that is independent of {VirtProductName} installation.
+You can use the Node Maintenance Operator to place nodes in maintenance mode.
 
 [NOTE]
 ====
-If you have installed {VirtProductName}, you must use the Node Maintenance Operator that is bundled with it.
+If {VirtProductName} v4.10.z or less is installed in your cluster, it includes an outdated version of the Node Maintenance Operator version. You must first upgrade {VirtProductName}, and then install the latest version of the Node Maintenance Operator from the `OperatorHub`.
```

Comment:
I would move the NOTE from the beginning to be under [Installing the Node Maintenance Operator](http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#installing-standalone-nmo) since it is about installing.

Reply:
By looking in https://issues.redhat.com/browse/CNV-17623?focusedCommentId=20673382&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20673382 I see that CNV suggest that a user can and should use "our" NMO before doing an upgrade if he/she want the node to stay under maintenance while doing the CNV upgrade...

I would suggest to leave the note, without an action from the user, as follows:


> If OpenShift Virtualization v4.10.z or less is installed in your cluster, it includes an outdated version of the Node Maintenance Operator version.

Reply:
Updated as requested http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#installing-standalone-nmo